guides/release/components/customizing-a-components-element.md

```diff
@@ -33,6 +34,36 @@ export default Component.extend({
 </ul>
 ```
 
+Note: It is not recommended to use null tagName .
+
+The following example explains usecase when `tagName` is null:
+
+When the `tagName` is assigned to null value, default `div` tag will be provided by ember but element bindings such as `classNameBindings` will not be available for the root element in component. 
+
+```javascript
```

Comment:
If you add {data-filename=app/components/my-component.js} after the word handlebars and the filename and type will show up in the Guides.